doc-content/jbpm-docs/src/main/asciidoc/CoreEngine/runtime-manager-strategies-con.adoc

```diff
@@ -0,0 +1,60 @@
+
+The same instance of `RuntimeEngine` must be used within single transaction to ensure correctness of state. Otherwise, an operation completed in one call would not be visible in the next call.
+
+This stragegy is "stateless", as process state is preserved only within the request. When a request is completed, the `RuntimeEngine` is permanently destroyed. If persistence is used, information related tothe KIE session is removed from the persistence database as well.
+
+The strategy has the following characteristics:
+
+* It provides completely isolated {PROCESS_ENGINE} and task service operations for every request.
+* It is completely stateless, storage of any facts happens only for the duration of the request.
+* It is a good fit for high-load, stateless processes, where no facts or timers must be preserved between requests.
+* The `KieSession` is only available during the lifetime of a request and is destroyed at the end of the request.
+* The strategy is not contextual. When you retrieve instances of RuntimeEngine from a singleton RuntimeManager, you do not need to take the `Context` instance into account. Usually, you can use EmptyContext.get() as the context, although a null argument is acceptable as well
+
+*Per process instance strategy*: instructs `RuntimeManager` to maintain a strict relationship between a KIE session and a process instance. Each `KieSession` is available as long as the `ProcessInstance` to which it belongs is active.
+
+This strategy provides the most flexible approach to use advanced capabilities of the {PROCESS_ENGINE} like rule evaluation and isolation between process instances. It maximizes performance and reduces potential bottlenecks introduced by synchronization. At the same time, unlike the request strategy, it reduces the number of KIE sessions to the actual number of process instances, rather than the total number of requests.
+
+This strategy has the following characteristics:
+
+* It provides isolation for every process instance.
+* It maintains a strict relationship between `KieSession` and `ProcessInstance` to ensure that it will always deliver the same `KieSession` for a given `ProcessInstance`.
+* It merges the lifecycle of `KieSession` with `ProcessInstance`, and both are disposed when the process instance completes or aborts.
+* It enables maintaining data, such facts and timers, in the scope of the process instance. Only the process instance has access to the data.
+* It introduces some overhead because of the need to look up and load the `KieSession` for the process instance.
+* It validates every usage of a `KieSession` so it cannot be used for other process instances. An exception is thrown if another process instance would use the same `KieSession`.
+* The strategy is contextual and accepts the following context instances:
+
+** `EmptyContext` or null: When starting a process instance, as there is no process instance id available yet
```

Comment:
Change to "Used when starting a process instance because no process ID is available yet"  Note cap ID. Also note that per IBM SG, avoid empty "there is/are".